packages/venia-ui/lib/util/formValidators.js

```diff
@@ -159,3 +159,19 @@ export const isNotEqualToField = (value, values, fieldKey) => {
     };
     return value !== values[fieldKey] ? SUCCESS : message;
 };
+
+export const isValidateEmail = value => {
+    const FAILURE = {
+        id: 'validation.isValidateEmail',
+        defaultMessage:
+            'Please enter a valid email address (Ex: johndoe@domain.com).'
+    };
+    if (
+        /^([a-z0-9,!\#\$%&'\*\+\/=\?\^_`\{\|\}~-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z0-9,!\#\$%&'\*\+\/=\?\^_`\{\|\}~-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*@([a-z0-9-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z0-9-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*\.(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]){2,})$/i.test(
```

Comment:
Please see here https://hackernoon.com/the-100-correct-way-to-validate-email-addresses-7c4818f24643

suggest to check for the presence of the @ symbol only.

Reply:
@fooman given link is not working (https://hackernoon.com/the-100-correct-way-to-validate-email-addresses-7c4818f24643) 
I had created email validation based on the current **Magento 2 Blank/Luma** theme validation, This covers almost all email address validation and This regular expression was validating Magento since Magento 2.0.x.

Reply:
sorry let's see if this works better https://medium.com/hackernoon/the-100-correct-way-to-validate-email-addresses-7c4818f24643

The gist of the article is that emails are so complicated that the best is to check that the user did enter something resembling an email address. Checking for the presence of the @ character achieves that goal. The only way to know for sure that an email address is indeed correct is to have the back-end send a validate your email mail.

Reply:
Probably Luma should be changed as well.